designer/client/outputs/notify-edit.tsx

```diff
@@ -113,6 +114,31 @@ class NotifyEdit extends Component<Props, State> {
           data={data}
           onEdit={onEdit}
         />
+        <div className="govuk-form-group">
+          <Checkboxes
+            items={[
+              {
+                children: (
+                  <strong>Include webhook and payment references</strong>
+                ),
+                hint: {
+                  children: (
+                    <div>
+                      If webhook or payment references are available, they will
+                      be included in Notify&apos;s personalisation object.
+                      <br />
+                      The included fields are: hasWebhookReference (boolean),
+                      webhookReference: (string), hasPaymentReference:
```

Comment:
I think we already provide webhookReference as just `reference` - I think it might cause some issues if some existing FCDO templates are already using reference

Reply:
Thanks @jenbutongit 

I thought I'd add that because the reference availability inside the template is neighther explict nor documented anywhere, reading notify's documentation it also seems that reference shouldn't be a 1-to-1 relationship with the user, it is more an identification of the message, which can be also be used for bulk emails. 

> A unique identifier you create. This reference identifies a single unique notification or a batch of notifications.
 https://docs.notifications.service.gov.uk/node.html#send-an-email-arguments-reference-required
 
To keep backward compatibility I decided not to include the webhookReference variable because this wouldn't affect the existing reference make things explicit.

I could remove it and keep just `hasReference`, `hasPaymentReference` and `paymentReference`. Thoughts?

Reply:
Thanks @jenbutongit 

I thought I'd add that because the reference availability inside the template is neither explicit nor documented anywhere (including notify's docs), reading notify's documentation it also seems that reference shouldn't be a 1-to-1 relationship with the user, it is more an identification of the message, which can also be used for bulk emails. 

> A unique identifier you create. This reference identifies a single unique notification or a batch of notifications.
 https://docs.notifications.service.gov.uk/node.html#send-an-email-arguments-reference-required
 
To keep backward compatibility I decided not to include the webhookReference variable because this wouldn't affect the existing reference and make things explicit.

We could just have `hasReference`, `hasPaymentReference` and `paymentReference`, although I'd rather have the new variable, because of the bulk emails and also to save time for users as I had to try and test to figure out the reference behaviour,  What do you think?

Reply:
ping @jenbutongit

Reply:
new variable sgtm. you'll just have to double check the existing templates then

Reply:
Perfect. Thank you!